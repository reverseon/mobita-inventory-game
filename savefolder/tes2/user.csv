id;username;nama;alamat;password;role
38070;thirafinajwan;Thirafi Najwan;Ehe;yahoooitspw;user
75844;something;something;something;something;user
