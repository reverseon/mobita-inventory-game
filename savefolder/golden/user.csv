id;username;nama;alamat;password;role
1;adminserver;Admin Server;none;admin123;admin
38070;thirafinajwan;Thirafi Najwan;Ehe;password123;user
75844;something;Ken Watanabe;something;something;user
53615;naotomori;Nao Tomori;naotomori12;naotomori;user
